src/main/java/uk/gov/companieshouse/officer/delta/processor/processor/DeltaProcessor.java

```diff
@@ -65,70 +60,54 @@ public void process(ChsDelta delta) throws RetryableErrorException, NonRetryable
             // ApiErrorResponseException.
-            logger.info("Failed to process officer delta", DataMapHolder.getLogMap());
+            LOGGER.info("Failed to process officer delta", DataMapHolder.getLogMap());
             throw new RetryableErrorException("Failed to send data for officer, retry", e);
         }
     }
 
     @Override
     public void processDelete(ChsDelta chsDelta) {
-        final String logContext = chsDelta.getContextId();
         OfficerDeleteDelta officersDelete;
         try {
             officersDelete = objectMapper.readValue(chsDelta.getData(),
                     OfficerDeleteDelta.class);
-            final String companyNumber = officersDelete.getCompanyNumber();
-            DataMapHolder.get()
-                    .companyNumber(companyNumber)
-                    .officerId(officersDelete.getOfficerId())
-                    .internalId(officersDelete.getInternalId());
-
-            final String internalId = TransformerUtils.encode(officersDelete.getInternalId());
-            apiClientService.deleteAppointment(logContext, internalId, companyNumber, officersDelete.getDeltaAt());
-        } catch (ResponseStatusException e) {
-            handleResponse(e, e.getStatus(), logContext, "Sending officer delete failed");
-        } catch (Exception ex) {
-            logger.error("Error when extracting officers delete delta", ex, DataMapHolder.getLogMap());
+        } catch (JsonProcessingException ex) {
+            LOGGER.error("Error when extracting officers delete delta", ex, DataMapHolder.getLogMap());
             throw new NonRetryableErrorException(
                     "Error when extracting officers delete delta", ex);
         }
-    }
 
-    private void handleResponse(final ResponseStatusException ex, final HttpStatus httpStatus, final String logContext,
-            final String msg)
-            throws NonRetryableErrorException, RetryableErrorException {
-        Map<String, Object> logMap = DataMapHolder.getLogMap();
-        logMap.put("status", httpStatus.toString());
-        if (HttpStatus.BAD_REQUEST == httpStatus || HttpStatus.CONFLICT == httpStatus) {
-            logger.errorContext(logContext, msg, ex, logMap);
-            throw new NonRetryableErrorException(msg, ex);
-        }
-        else {
-            // any other client or server status is retryable
-            logger.infoContext(logContext, msg + ", retry", logMap);
-            throw new RetryableErrorException(msg, ex);
-        }
+        final String companyNumber = officersDelete.getCompanyNumber();
+        final String officerId = officersDelete.getOfficerId();
+        final String internalId = officersDelete.getInternalId();
+        DataMapHolder.get()
+                .companyNumber(companyNumber)
+                .officerId(officerId)
+                .internalId(internalId);
+
+        final String encodedInternalId = TransformerUtils.encode(internalId);
```

Comment:
Could also add `encodedInternalId` to the log map as `DataMapHolder.get().appointmentId(encodedInternalId);`. The same can also be done in `AppointmentTransform` during the upsert process.